Sofa/framework/Core/src/sofa/core/behavior/BaseMatrixLinearSystem.cpp

```diff
@@ -27,15 +27,22 @@ namespace sofa::core::behavior
 BaseMatrixLinearSystem::BaseMatrixLinearSystem()
 : Inherit1()
 , d_matrixSize(initData(&d_matrixSize, "matrixSize", "Size of the global matrix"))
+, d_authorizeAssembly(initData(&d_authorizeAssembly, true, "authorizeAssembly", "Allows to assemble the system matrix"))
```

Comment:
```suggestion
, d_authorizeAssembly(initData(&d_authorizeAssembly, true, "authorizeAssembly", "Enable assembly of the system matrix"))
```
and maybe renaming the Data with d_enableAssembly (`authorize` seems ... too strong ?)

Reply:
This applies to the other recurrences of this `d_authorizeAssembly` Data in other components

Reply:
done